src/app/app.component.ts

```diff
@@ -9,17 +16,56 @@ import { Router } from '@angular/router';
 export class AppComponent {
   title = 'splitwise';
   // Declare showLoginButton property
-  showLoginButton: boolean;
+  showLogin: boolean;
+  isSideNavCollapsed = false;
```

Comment:
Again variable initialization and declaration rule not followed

Reply:
Change done.